app/scripts/components/exploration/components/timeline/timeline-controls.tsx

```diff
@@ -228,8 +236,27 @@ export const TimelineHeadIndicators = memo(({ outOfViewHeads }: { outOfViewHeads
 
 TimelineHeadIndicators.displayName = 'TimelineHeadIndicators';
 
+/**
+ * Determines the appropriate calendar view based on the time density.
+ *
+ * The TimeDensity enumeration is mapped to the corresponding calendar view:
+ * - MONTH: Displays the calendar in 'year' view, showing all months in a year.
+ * - YEAR: Displays the calendar in 'decade' view, showing multiple years in a decade.
+ * - Default: Displays the calendar in 'month' view, showing all days of the current month.
+ */
+const getCalendarView = (timeDensity: TimeDensity): View | undefined => {
+  switch (timeDensity) {
```

Comment:
⛏️ : can get rid of `undefined` return type? Because it looks like it would default to `month` string